components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/resources/management-api.yaml

```diff
@@ -0,0 +1,1957 @@
+                    items:
+                      type: object
+                      properties:
+                        name:
+                          type: string
+                          example: endpointTemplate
+      security:
+        - BearerAuth: []
+    post:
+      summary: Enable or Disable Message Tracing for Sequence Templates
+      description: Enable or disable message tracing for a specified sequence template.
+      tags:
+        - Templates
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                name:
+                  type: string
+                  description: The name of the sequence template to enable or disable tracing.
+                  example: testSequenceTemplate
+                type:
+                  type: string
+                  description: The type of the template to enable or disable tracing.
+                  example: sequence
+                trace:
+                  type: string
+                  enum: [enable, disable]
+                  description: The action to perform for tracing.
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Enabled tracing for ('testSequenceTemplate')
+      security:
+        - BearerAuth: []
+  /server:
+    get:
+      summary: Get server information
+      description: Retrieves information related to the micro integrator server instance.
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  productVersion:
+                    type: string
+                    example: "4.2.0"
+                  osVersion:
+                    type: string
+                    example: "13.2.1"
+                  javaVersion:
+                    type: string
+                    example: "11.0.18"
+                  carbonHome:
+                    type: string
+                    example: "/Users/dedunukarunarathne/Documents/MI"
+                  javaVendor:
+                    type: string
+                    example: "Eclipse Adoptium"
+                  osName:
+                    type: string
+                    example: "Mac OS X"
+                  productName:
+                    type: string
+                    example: "WSO2 Micro Integrator"
+                  javaHome:
+                    type: string
+                    example: "/Library/Java/JavaVirtualMachines/temurin-11.jdk/Contents/Home"
+      security:
+        - BearerAuth: []
+      tags:
+        - Server Information
```

Comment:
Shall we combine Server Information with Server Management

Reply:
Done with d15dd7b1ed547da8ba7d23a10d9eeee1789dd9d5